bokehjs/examples/anscombe/anscombe.ts

```diff
@@ -1,8 +1,8 @@
 namespace Anscombe {
   console.log(`Bokeh ${Bokeh.version}`);
-  Bokeh.set_log_level("debug");
+  Bokeh.set_log_level("info");
 
-  const anscombe_quartet = Bokeh.LinAlg.transpose([
+  const anscombe_quartet = Bokeh.Utils.transpose([
```

Comment:
Then there is no point in changing it from `LinAlg` in this PR. A PR should have both a decision and a change, or neither.